packages/webapp/webapp_server.js

```diff
@@ -500,8 +500,12 @@ var runWebAppServer = function () {
         var program = {
           format: "web-program-pre1",
           manifest: manifest,
-          version: WebAppHashing.calculateClientHash(manifest, null, _.pick(
-            __meteor_runtime_config__, 'PUBLIC_SETTINGS')),
+          version: process.env.AUTOUPDATE_VERSION ||
```

Comment:
Ah, @abernix tells me it's not a new variable, so we should probably keep it as-is.